src/main/scala/verifier/DefaultMainVerifier.scala

```diff
@@ -303,6 +303,21 @@ class DefaultMainVerifier(config: Config,
       case r => r
     }
 
+    val mce = member.info.getUniqueInfo[ast.AnnotationInfo] match {
+      case Some(ai) if ai.values.contains("exhaleMode") =>
+        ai.values("exhaleMode") match {
+          case Seq("0") | Seq("greedy") =>
+            if (Verifier.config.counterexample.isSupplied)
+              reporter report AnnotationWarning(s"Member ${member.name} has exhaleMode annotation that may interfere with counterexample generation.")
+            false
+          case Seq("1") | Seq("mce") | Seq("moreCompleteExhale") => true
+          case v =>
+            reporter report AnnotationWarning(s"Member ${member.name} has invalid exhaleMode annotation value $v. Annotation will be ignored.")
```

Comment:
Mh I'm not sure. A consistency check is an AST-level check that's usually independent of the backend, right, and this is a backend-specific annotation. So we'd either need to build in checks regarding what a correct annotation is right into the AnnotationInfo AST node, which seems wrong to me, since annotations should in principle allow you to add all kinds of stuff in a flexible way. Or it wouldn't be a consistency check and we'd return an AnnotationError here instead, so you wouldn't be able to ignore it.
This might actually be something we should discuss with more people, since this will come up again and again.

Reply:
Yeah I can see that. Actually the more I think about it the more a backend-specific consistency check would also make sense. But I think this is just something we need to discuss and decide once and for all, for annotations in general. I think it's not super clear what's best. IMO warnings *could* also make sense:
- We at some point had the idea that it should always be sound to ignore an annotation.
- We can't reliably flag errors for all potential problems here. E.g. we would detect ``@exhaleMode("nce")`` but not ``@exhaleNode("mce")``. For the latter we can't flag an error (at most a warning), it's just an annotation we don't recognize, and we can't flag errors for all annotations we don't recognize since otherwise Carbon would have to reject all programs with Silicon-specific annotations.

So I'd suggest going with the warning for now and making sure we discuss this asap. Changing this to an error would be super quick and simple of course.